resource-recorder/migrations/0000_init.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS resources (
+    project_id TEXT,                                -- Identifier of the project this resource belongs to.
+    service_id TEXT,                                -- Identifier of the service this resource belongs to.
+    type TEXT,                                      -- Type of resource this is.
+    data TEXT,                                      -- Data about this resource.
+    config TEXT,                                    -- The config to create the object for this resource.
+    is_active boolean,                              -- The config to create the object for this resource.
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP, -- Time this resource was created.
+    PRIMARY KEY (project_id, service_id, type)
```

Comment:
I recognize this as the second NSS proposal of your URN rfc, out of curiosity, does that mean we will only use this or will we also use the one with names?

Reply:
At this level we will use the id one. But I am hoping we will be able to allow users to use the names one, then gateway + deployer should have enough information to convert the names one to this id one.